.github/workflows/ci.yml

```diff
@@ -25,20 +25,20 @@ jobs:
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v1
     with:
       envs: |
-        - linux: test-oldestdeps-cov-xdist
+        - linux: test-oldestdeps-cov
           python-version: 3.9
-        - linux: test-xdist
+        - linux: test
           python-version: 3.9
-        - linux: test-xdist
+        - linux: test
           python-version: 3.10
-        - linux: test-xdist
+        - linux: test
           python-version: 3.11
-        - macos: test-xdist
+        - macos: test
           python-version: 3.11
-        - linux: test-cov-xdist
+        - linux: test-cov
```

Comment:
Could you include names (ie Python 3.9) for all these since you are updating it and the branch protections anyways? This would make the side bar in the CI easier to use.